{{cookiecutter.repo_name}}/README.txt

```diff
@@ -20,10 +20,22 @@ Getting Started
 
     env/bin/pip install -e ".[testing]"
 
-- Configure the database.
+- Use alembic to generate revisions before you run initializedb. Example:
+
+    env/bin/alembic -c development.ini revision --autogenerate -m "init"
+
+- Then you can use alembic to upgrade to that revision:
+
+    env/bin/alembic -c development.ini upgrade head
+
+- Or initialize the database.
 
     env/bin/initialize_{{ cookiecutter.repo_name }}_db development.ini
 
+- Run database migrations.
+
+    env/bin/alembic -c development.ini ...
+
```

Comment:
The instructions must be consistent throughout.

Also this is the only set of instructions where you indicate that the user can run migrations. Instructions should align with typical workflows.